src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -20,12 +24,21 @@ public void initialize() {
 		// Configure operator command chooser
 		operatorChooser.addDefault(new DefaultOperator());
 		// Initialize SmartDashboard display values
-		// SmartDashboard.putNumber(SmartDashboardKey.EXAMPLE.key, 0);
+		SmartDashboard.putNumber("LIDAR_P", Lidar.LIDAR_TURN_P);
+		SmartDashboard.putNumber("LIDAR_I", Lidar.LIDAR_TURN_I);
+		SmartDashboard.putNumber("LIDAR_D", Lidar.LIDAR_TURN_D);
 	}
 	
 	@Override
 	public void teleopInitialize() {
 		teleopCommand = new Noop();
+		LidarCommand = new LidarTurner();
+		LidarCommand.start();
+		RobotMap.Component.lidarMC.reset();
+		Lidar.LIDAR_TURN_P = SmartDashboard.getNumber("LIDAR_P", Lidar.LIDAR_TURN_P);
+		Lidar.LIDAR_TURN_I = SmartDashboard.getNumber("LIDAR_I", Lidar.LIDAR_TURN_I);
+		Lidar.LIDAR_TURN_D = SmartDashboard.getNumber("LIDAR_D", Lidar.LIDAR_TURN_D);
+		RobotMap.Component.lidarMC.setPID(Lidar.LIDAR_TURN_P, Lidar.LIDAR_TURN_I, Lidar.LIDAR_TURN_D);
```

Comment:
In my opinion, we should remove the SmartDashboard code now that testing has finished.